blockmanager_test.go

```diff
@@ -70,7 +70,7 @@ func setupBlockManager() (*blockManager, headerfs.BlockHeaderStore,
 	}
 
 	// Set up a blockManager with the chain service we defined.
-	bm, err := newBlockManager(cs)
+	bm, err := newBlockManager(cs, nil)
```

Comment:
We don't yet use `blockManager.Start()` anywhere within the tests, but if we do, this will block startup. Should probably add a `nil` check there to avoid this.